python/lsst/sims/photUtils/SignalToNoise.py

```diff
@@ -7,22 +7,45 @@
           "calcM5", "calcSkyCountsPerPixelForM5", "calcGamma", "calcSNR_m5",
           "calcAstrometricError", "magErrorFromSNR", "calcMagError_m5", "calcMagError_sed"]
 
+def FWHMeff2FWHMgeom(FWHMeff):
+    """
+    @param [in] FWHMeff (the single-gaussian equivalent FWHM value, appropriate for calcNeff) in arcseconds
 
-def calcNeff(seeing, platescale):
+    @param [out] FWHMgeom (the geometric FWHM value, as measured from a typical PSF profile) in arcseconds
     """
-    Calculate the effective number of pixels in a double Gaussian PSF
+    FWHMgeom = 0.822*FWHMeff + 0.052
+    return FWHMgeom
 
-    @param [in] seeing in arcseconds
+def FWHMgeom2FWHMeff(FWHMgeom):
+    """
+    @param [in] FWHMgeom (the geometric FWHM value, as measured from a typical PSF profile) in arcseconds
+
+    @param [out] FWHMeff (the single-gaussian equivalent FWHM value, appropriate for calcNeff) in arcseconds
+    """
+    FWHMeff = (FWHMgeom - 0.052)/0.822
+    return FWHMeff
+
+def calcNeff(FWHMeff, platescale):
+    """
+    Calculate the effective number of pixels in a single gaussian PSF
+
+    @param [in] FWHMeff in arcseconds
+       (the width of a single-gaussian that produces correct Neff for typical PSF profile)
 
     @param [in] platescale in arcseconds per pixel
 
-    @param [out] the effective number of pixels contained by a double
-    Gaussian PSF
+    @param [out] the effective number of pixels contained in the PSF
 
     see equation 31 of the SNR document
     https://docushare.lsstcorp.org/docushare/dsweb/ImageStoreViewer/LSE-40
+
+    The FWHMeff is a way to represent the equivalent seeing value, if the
+    atmosphere could be simply represented as a single gaussian (instead of a more
+    complicated von Karman profile for the atmosphere, convolved properly with the
+    telescope hardware additional blurring of 0.4").
+    A translation from the geometric FWHM to the FWHMeff is provided in FWHMgeom2FWHMeff.
     """
-    return 2.436*(seeing/platescale)**2
+    return 2.266*(FWHMeff/platescale)**2
```

Comment:
Is the SNR document being updated so that equation (31) now agrees with this expression?  I ask because the docstring claims that this expression comes from equation (31) of the SNR document.